NEWS

```diff
@@ -353,6 +353,15 @@ CHANGES WITH 233 in spe
 
 CHANGES WITH 232:
 
+        * udev now runs with MemoryDenyWriteExecute=, RestrictRealtime= and
+          RestrictAddressFamilies= enabled. These sandboxing options should
+          generally be compatible with the various external udev call-out
+          binaries we are aware of, however there may be exceptions, in
+          particular when exotic languages for these call-outs are used. In
```

Comment:
> when exotic languages for these call-outs are used

@poettering , some people use [`pypy`](https://pypy.org/). And `pypy` fails with
```
Got an unexpected error trying to allocate some memory for the JIT (tried to do mmap() with PROT_EXEC|PROT_READ|PROT_WRITE)
```
I think `Python` is not an exotic language. The same for `luajit`, `node` and so on.

Regarding #5400 and
> if compatibility with exotic udev call-outs is a priority, and security is not

Actaully, people build `PIE`-binaries for security reasons (see https://fedoraproject.org/wiki/Changes/golang-buildmode-pie)

Anyway, I think people know how to find `man systemd.exec`
> Note that this option is incompatible with programs that generate program code dynamically at runtime, such as JIT
           execution engines, or programs compiled making use of the code "trampoline" feature of various C compilers